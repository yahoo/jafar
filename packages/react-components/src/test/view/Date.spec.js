/**
  * Copyright 2020, Verizon Media
  * Licensed under the terms of the MIT license. See LICENSE file in project root for terms.
  */

import React from 'react';
import { shallow } from 'enzyme';
import DateView from '../../components/view/Date';

describe('<Date />', () => {
  let component;
  const value = new Date('Thu, 16 Jan 2020 08:27:07 GMT');

  beforeEach(() => {
    component = shallow(
      getComponent(value)
    );
  });
  it('should render provided data', () => {
    expect(component).toMatchSnapshot();
  });

  function getComponent(value) {
    return (
      <DateView value={value} />
    );
  }
});
