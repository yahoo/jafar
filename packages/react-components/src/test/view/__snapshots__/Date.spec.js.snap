// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Date /> should render provided data 1`] = `
<div>
  01/16/2020
</div>
`;
